fastlane/lanes/release.rb

```diff
@@ -280,3 +290,15 @@ def report_milestone_error(error_title:)
 
   buildkite_annotate(style: 'warning', context: 'error-with-milestone', message: error_message) if is_ci
 end
+
+def delete_all_metadata_release_notes(store_metadata_folder: STORE_METADATA_FOLDER)
+  pattern = File.join(store_metadata_folder, '**', 'release_notes.txt')
+  Dir[pattern].each { |path| File.delete(path) }
+  git_add(path: pattern, shell_escape: true)
+  git_commit(
+    path: pattern,
+    message: 'Delete release notes source and localization before code freeze',
+    # Even if no locale was translated in the previous cycle, default/relaese_notes.txt should always be present, and therefore deleted at this stage.
+    allow_nothing_to_commit: false
+  )
+end
```

Comment:
wdyt of this instead, to avoid using the pattern itself in the git commands, and only use them for the `glob` instead?

```suggestion
def delete_all_metadata_release_notes(store_metadata_folder: STORE_METADATA_FOLDER)
  files = Dir.glob(File.join(store_metadata_folder, '**', 'release_notes.txt'))
  files.each { |path| File.delete(path) }
  git_add(path: files)
  git_commit(
    path: files,
    message: 'Delete release notes source and localization before code freeze',
    # Even if no locale was translated in the previous cycle, default/relaese_notes.txt should always be present, and therefore deleted at this stage.
    allow_nothing_to_commit: false
  )
end
```

Reply:
Sounds good!